doc/style_guide.md

```diff
@@ -163,8 +163,9 @@ Godot usage
   (`?`). The content of the popup should give more details and also
   end with a question.
 
-- Popups should be shown with `PopupCenteredMinsize()` unless there's
-  a good reason why something else is required.
+- Popups should be shown with `PopupCentered()` unless there's
+  a good reason why something else is required. To shrink the popup
+  to a minimum size, use the `PopupCenteredShrink()` extension method.
```

Comment:
I think this should be worded so that PopupCenteredShrink is the optimal, PopupCentered is the second best choice and then anything else besides those need a good reason why they are used.